.editorconfig

```diff
@@ -12,3 +12,7 @@ insert_final_newline = true
 [*.yml]
 indent_style = space
 indent_size = 2
+
+[*.java]
+ij_java_names_count_to_use_import_on_demand = 999
+ij_java_class_count_to_use_import_on_demand = 999
```

Comment:
Why is this change necessary?

Reply:
As the commit message says: this instructs the IDE to not try to bundle multiple imports into one with a star (default behavior in Intellij IDEA) - this seems to be the repo's preference. Admittedly it has no direct connection with the feature but keeps you from messing up when you use automatic code formatting.

Reply:
That is fine. The checkstyle rules would prevent using asterisks for multiple imports.